frontend/src/components/MeetingRoom/ToolbarOverflowMenu/ToolbarOverflowMenu.tsx

```diff
@@ -92,19 +100,22 @@ const ToolbarOverflowMenu = ({
                   setIsEmojiGridOpen={setIsEmojiGridOpen}
                   isParentOpen={isToolbarOverflowMenuOpen}
                 />
-                <ArchivingButton />
+                <ArchivingButton handleCloseMenu={handleCloseMenu} />
```

Comment:
I would call this prop `handleClick`  and the action `closeMenu`

Reply:
the wrapper isn't being called here, I think you can skip the wrapper in this one and just set it as`closeMenu` directly.